_posts/2015-12-24-two-column.markdown

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: 2 column Blog with Video
```

Comment:
Hello and thanks for the PR! The title should correspond to the filename therefore You should change title to `Two Column` or rename the post file to `2015-12-24-2-column-blog-with-video.markdown`.